repo/packages/Z/zeppelin/0/package.json

```diff
@@ -3,7 +3,7 @@
   "name": "zeppelin",
```

Comment:
@surdy unrelated to your changes, but `packagingVersion` 2.0 is not supported anymore and the PR tests fail on that. Could you update that for `Zeppelin` packages ?

Reply:
If that's going to block this PR then I would suggest either 
1) Removing the packages from repo altogether
2) I can modify this PR to not touch the 2.0 packages and the owner of the Universe repo can update the packages that are still using 2.0

Reply:
lets for `2`